### PR TITLE
add pythia configs for additional displaced SUSY decay (Run II)

### PR DESCRIPTION
**Pull request by jalimena**
Add stop to l+d decays to displaced SUSY signal MC (for EXO). These pythia8 configs are for Run II (2017).

### CODE REVIEW

File: python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py [hunk 1]
```diff
@@ -0,0 +1,857 @@
...
+##   and a zero mass might be hardwired in the model definition.   *
+##                                                                 *
+##       The SUSY decays have calculated using SDECAY 1.1a         *
+##                                                                 *
+##******************************************************************
+#
+BLOCK DCINFO  # Decay Program information
+     1   SDECAY      # decay calculator
+     2   1.1a        # version number
+#
```
Comment (efeyazgan) [inline python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py]: please delete such lines

File: python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py [hunk 2]
```diff
@@ -0,0 +1,857 @@
...
+        parameterSets = cms.vstring(
+            'pythia8CommonSettings',
+            'pythia8CP5Settings',
+            'processParameters'
+        )
+    )
+)
+
+dirhadrongenfilter = cms.EDFilter("MCParticlePairFilter",
+    #Status = cms.untracked.vint32(1, 1),
```
Comment (efeyazgan) [inline python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py]: please delete this commented line

File: python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py (hunk 1 above)
Comment (jalimena) [reply python/ThirteenTeV/DisplacedSUSY/DisplacedSUSY_stopToLD_M_XXX_YYYmm_TuneCP5_13TeV_pythia8_cff.py]: sorry, can you tell me exactly which lines? it's not clear to me from github. L43-46 inclusive? thanks!

**Comment by jalimena** [comment]
ok done, thanks for explaining!

**Pull request merged**